lib/ClangImporter/ClangImporter.cpp

```diff
@@ -4102,7 +4104,7 @@ void ClangModuleUnit::lookupAvailabilityDomains(
   if (domainInfo.Kind == clang::FeatureAvailKind::None)
     return;
 
-  auto *varDecl = dyn_cast_or_null<clang::VarDecl>(domainInfo.Decl);
+  clang::VarDecl *varDecl = nullptr;
```

Comment:
Is this change intended?

Reply:
This is temporary. I will ditch the "work around missing commit" commits as soon as the cherry-picks land on the new stable branch.